### ISSUE TITLE
Attempting to use the legacy API results in an error after upgrading to latest release

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#673]
In GitLab by @jauerb on Jan 30, 2019, 10:29

*Please describe how you installed Flake8*

```
$ pip install flake8
```

*Please provide the exact, unmodified output of `flake8 --bug-report`*

```
$ flake8 --bug-report
{
  "dependencies": [
    {
      "dependency": "entrypoints",
      "version": "0.3"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.6.3",
    "system": "Linux"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "flake8-docstrings",
      "version": "1.3.0, pydocstyle: 2.1.1"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "naming",
      "version": "0.8.0"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.5.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.1.0"
    }
  ],
  "version": "3.7.1"
}
```

*Please describe the problem or feature*

Attempting to use the legacy API results in an error after upgrading to latest release

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*

```
>>> from flake8.api import legacy as flake8
>>> style_guide = flake8.get_style_guide()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jauerbach/anaconda3/lib/python3.6/site-packages/flake8/api/legacy.py", line 49, in get_style_guide
    application.make_notifier()
AttributeError: 'Application' object has no attribute 'make_notifier'
```

This was working fine previously.

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 10:33

Confirmed, thanks for the report.

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 11:01

mentioned in merge request !293

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 11:06

closed via merge request !293

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 11:06

mentioned in commit 077b640a86e87719bc8cd751f4f967564e044633

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 12:53

This has been released in 3.7.2 -- thanks again for the issue :tada:

**Comment by asottile** [comment]
In GitLab by @jauerb on Jan 30, 2019, 13:06

Thanks so much for addressing this so quickly!!!!  👏👏👏👏👏
